tensorrt_llm/commands/build.py

```diff
@@ -139,7 +153,13 @@ def parse_arguments():
     return args
 
 
-def build_model(model: PretrainedModel, build_config: BuildConfig) -> Engine:
+def build_model(rank: int,
```

Comment:
The `build_model` is moved to `builder.py`, and renamed to `build` function.

We want to keep the interface of `build` function to be stable, 

`def build(model: PretrainedModel, build_config: BuildConfig) -> Engine:`

So, I suggest we add the `dry_run` and `visualize_network` as a field of `BuildConfig`

Reply:
Updated PR to add `dry_run` and `visualize_network` to `BuildConfig` as suggested.